templates/project/package.json

```diff
@@ -11,7 +11,8 @@
     "lint": "eslint --ext .ts src"
   },
   "dependencies": {
-    "{{BAZED_PACKAGE}}": "^{{BAZED_VERSION}}"
+    "{{BAZED_PACKAGE}}": "^{{BAZED_VERSION}}",
+    "zod": "^3.22.4"
   },
   "devDependencies": {
     "@types/node": "^20.5.0",
```

Comment:
Same here, `"@types/node": "^20.5.0"`, also, `engines` should be changed in the template project to acomodate for Node 16.